src/hadoop-data-node/config/hadoop-data-node.md

```diff
@@ -0,0 +1,44 @@
+## Hadoop data node section parser
+
```

Comment:
Pls link from below doc data path config to this file

https://github.com/Microsoft/pai/blob/master/docs/pai-management/doc/how-to-generate-cluster-config.md#optional-step-3-customize-configure-openpai-

Reply:
the problem is the same with below, if this doc is just for config. Later could tell user config info at service-config doc or other part